root/entrypoint.sh

```diff
@@ -1,27 +1,105 @@
 #!/bin/sh
+# Exit when any command fails
+set -e
 
-provisioner () {
-    echo "Starting provisioner..."
-    if ! out=`ansible-playbook -i /etc/ansible/hosts /etc/ansible/entrypoint.yml -c local "$@"`;then
-        echo $out;
+COMMAND_ARG1="$1"
+COMMAND_ARG2="$2"
+
+cd /var/www/wallabag || exit
+
+wait_for_database() {
+    timeout 60s /bin/sh -c "$(cat << EOF
+        until echo 'Waiting for database ...' \
+            && nc -z ${SYMFONY__ENV__DATABASE_HOST} ${SYMFONY__ENV__DATABASE_PORT} < /dev/null > /dev/null 2>&1 ; \
+        do sleep 1 ; done
+EOF
+)"
+}
+
+install_wallabag() {
+    su -c "php bin/console wallabag:install --env=prod -n" -s /bin/sh nobody
+}
+
+provisioner() {
+    SYMFONY__ENV__DATABASE_DRIVER=${SYMFONY__ENV__DATABASE_DRIVER:-pdo_sqlite}
+    POPULATE_DATABASE=${POPULATE_DATABASE:-True}
```

Comment:
I don't know if we want to have POPULATE_DATABASE set to true by default.

On a side-note I would be for replacing T with t as it is a Python thing. (`True` => `true` in conditions below)

Reply:
I didn't change the behavior in this PR, Ansible install set the same value. I think this en var is not well documented. Maybe we could improve this in other PR but for now I don't want to break things for the users. Or can open an issue to keep the track...

Reply:
Noted